staging/src/k8s.io/kubectl/pkg/cmd/logs/logs_test.go

```diff
@@ -374,6 +374,20 @@ func TestLog(t *testing.T) {
 			},
 			expectedErr: "Error from the ConsumeRequestFn",
 		},
+		{
+			name: "fail if no matching request exists",
```

Comment:
You are not passing the label selector here, is this a valid test case?

Reply:
@bseenu 
It covers all cases when kube-api server returns empty resources with no error.